models/public/mask_rcnn_inception_v2_coco/mask_rcnn_inception_v2_coco.md

```diff
@@ -71,7 +71,7 @@ bounding boxes. For each detection, the description has the format:
     - `conf` - confidence for the predicted class
     - (`x_min`, `y_min`) - coordinates of the top left bounding box corner (coordinates stored in normalized format, in range [0, 1])
     - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner  (coordinates stored in normalized format, in range [0, 1])
-2. name: `masks`
+2. Segmentation mask, name: `masks`, shape: [N, 90, 33, 33], where N is the number of detected masks. 
```

Comment:
is mask size 33 x 33?
As I remember the standard size is 28 x 28

Reply:
This is what i got after conversion